src/subscriptiondata.h

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef SUBSCRIPTIONDATA_H
+#define SUBSCRIPTIONDATA_H
+
+#include <QObject>
+
+class SubscriptionData final : public QObject {
+  Q_OBJECT
+  Q_DISABLE_COPY_MOVE(SubscriptionData)
+
+  Q_PROPERTY(bool initialized READ initialized NOTIFY changed)
+
+  // Subscription
+  Q_PROPERTY(int createdAt READ createdAt CONSTANT)
```

Comment:
if you don't need to read this value from C++, you can do:
`Q_PROPERTY(int createdAt MEMBER m_createdAt CONSTANT)`

Reply:
Changed for all the properties.